pkg/controller/traceflow/controller.go

```diff
@@ -265,6 +265,11 @@ func (c *Controller) syncTraceflow(traceflowName string) error {
 }
 
 func (c *Controller) startTraceflow(tf *crdv1alpha1.Traceflow) error {
+	if err := c.validateTraceflow(tf); err != nil {
+		klog.Errorf("Invalid traceflow request %+v, err: %+v", tf, err)
+		err = c.updateTraceflowStatus(tf, crdv1alpha1.Failed, "", 0)
```

Comment:
why do we always use an empty reason string?

Reply:
Sorry. It's my miss. It should be the same string we used in Line 269.